@xen-orchestra/lite/src/App/TabConsole.tsx

```diff
@@ -65,6 +68,27 @@ const TabConsole = withState<State, Props, Effects, Computed, ParentState, Paren
         </p>
       ) : (
         <>
+          <IntlMessage id='VmStartLabel' defaultMessage='loading'>
+            {VmStartLabel => (
+              <PanelHeader
+                actions={[
+                  {
+                    icon: 'play',
+                    color: 'primary',
+                    title:
```

Comment:
You can make it simpler like this:

https://github.com/vatesfr/xen-orchestra/blob/9abd1429a27b75939ab92789894fbb65a684da07/%40xen-orchestra/lite/src/App/TreeView.tsx#L69

Reply:
the title property want a `string` , but IntlMessage is an `Element`

Reply:
What do you think about managing the tooltip in `PanelHeader` component instead of using the title?

https://github.com/vatesfr/xen-orchestra/blob/9abd1429a27b75939ab92789894fbb65a684da07/%40xen-orchestra/lite/src/components/Tree.tsx#L19

https://github.com/vatesfr/xen-orchestra/blob/9abd1429a27b75939ab92789894fbb65a684da07/%40xen-orchestra/lite/src/components/Tree.tsx#L85